pages/solidity-api/ArxProjectEnrollmentManager.md

```diff
@@ -82,7 +82,9 @@ constructor(contract IChipRegistry _chipRegistry, contract ITSMRegistrar _tsmReg
 function addProject(address _projectManager, string _projectClaimDataUri, bytes32 _nameHash, bytes32 _merkleRoot, address _projectPublicKey, address _provingChip, struct IChipRegistry.TSMMerkleInfo _tsmMerkleInfo, struct IChipRegistry.ManufacturerValidation _manufacturerValidation, bytes _chipOwnershipProof, bytes _projectOwnershipProof) public
 ```
 
-Adds a new TSM's project to the ERS system by deploying the ProjectRegistrar contract via CREATE2 and
+// TODO: what is CREATE2 exactly? I'm still unclear on this, can we add a link to their documentation? edit: did I get the link right?
+// The TSM doesn't have to be new right?
+Adds a TSM's new project to the ERS system by deploying the ProjectRegistrar contract via [CREATE2](https://docs.openzeppelin.com/cli/2.8/deploying-with-create2) and
 registering it to the TSM Registrar. We use CREATE2 because we need the projectManager to provide proof of
```

Comment:
Switched wording from `a new TSM's project` to `a TSMs new project`, is this correct? the TSM doesn't have to be new

Reply:
![Screenshot 2023-12-22 at 8 23 47 PM](https://github.com/arx-research/ers-docs/assets/25701682/9c0b93db-47bf-40c8-814d-4f717e3de88b)
Was unsure which in this diagram, this `ArxProjectEnrollmentManager` is.